### ISSUE TITLE
Gui pro vlastní bod na mapě (custom marker)

### ISSUE THREAD

**Issue by mkyral** [osmcz/osmcz#80]
> btw, takový offtopic - na mapě nám funguje i URL pro custom marker:
> http://openstreetmap.cz/?mlat=50.79&mlon=15.16&zoom=17

Chybí tomu gui.

**Comment by mcepl** [comment]
No, ale je to takové divné. Když bych chtěl turistickou mapu s EU turistickými značkami (jaký je rozdíl mezi EU a ČR), tak jsem zkusil tuto velmi komplikovanou věc (co mi udělal Firefox v URL řádku)
`https://openstreetmap.cz/?mlat=50.486516&mlon=14.428733&zoom=16&layers=kH#map=16/50.486516/14.428733layers=kH`, ale to mi dalo Mapbox list bez turistických značek. Jak to všechno zkombinovat?

**Comment by mkyral** [comment]
Divné, divné. Tohle parser nějak nepobere. Bude potřeba opravit.

Rozdíl těch vrstev s turistickými značkami je v poskytovateli - ČR generuje Petr na poloha.net a je to klasické zobrazení tak jak jsme v ČR zvyklí a je omezeno pouze na ČR. EU je z waymarkedtrails.org a tam je to zase renderováno jinak (dvě barvy pro trasy plus symboly) a generuje se pro celou evropu.
